Resources/Private/Templates/PluginPreview/Events.html

```diff
@@ -96,7 +97,7 @@ <h4>Events2</h4>
     <f:if condition="{categories}">
         <tr>
             <td><strong>Categories</strong></td>
-            <td>{categories}</td>
+            <td>{e2:settingsFormat(content: '{categories}', glue: ',')}</td>
```

Comment:
curly braces are only need, if you want to process a ViewHelper, but as you just access a variable, you can remove quotes and braces here